integration/src/test/java/pages/CapabilitiesPage.java

```diff
@@ -0,0 +1,39 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class CapabilitiesPage extends PageObject{
+
+    @FindBy(id = "capTitle") private WebElement Title;
```

Comment:
Variables should be in camelCase
```suggestion
    @FindBy(id = "capTitle") private WebElement title;
```